src/redux/store.js

```diff
@@ -1,22 +1,14 @@
 import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import { createMiddleware as createAnalyticsMiddleware } from 'redux-beacon';
-import { GoogleAnalytics } from 'redux-beacon/targets/google-analytics';
```

Comment:
We shuoldn't delete anything of this, for this PR doesn't mather, but just for you to know

Reply:
okey!